contracts/modules/oracle/Oracle.sol

```diff
@@ -11,24 +11,63 @@ import "../../interfaces/IOracle.sol";
  *          - only makes request for USD prices and returns results in standard 8 decimals for Chainlink USD feeds
  */
 contract Oracle is IOracle {
+    /// @notice registry - Chainlink Feed Registry with aggregated prices across
     FeedRegistryInterface internal registry;
+    /// @notice NULL_PRICE - null price when asset price feed is deemed invalid
+    int256 public constant NULL_PRICE = 0;
+    /// @notice PRICE_DECIMALS - the normalized amount of decimals for returned prices in USD
+    uint8 public constant PRICE_DECIMALS = 8;
+    /// @notice MAX_PRICE_LATENCY - amount of time between oracle responses until an asset is determined toxiz
+    /// Assumes Chainlink updates price minimum of once every 24hrs and 1 hour buffer for network issues
+    uint256 public constant MAX_PRICE_LATENCY = 25 hours;
+
+    event StalePrice(address indexed token);
```

Comment:
@Craigson add `answerTimestamp` to events too so we know when price was last valid